linux-functions.sh

```diff
@@ -1,310 +0,0 @@
-#!/bin/sh
-# This is largely a copy of linux-setup.sh
-# TODO(ericbrown): Reduce code redundancy by refactoring to share code
-
-install_java() {
```

Comment:
I don't see where these got moved to, where they redundant?

**edit** Ok, cool, most if it was already in `linux-setup.sh`, and most of the important bits that wasn't was ported there. Looks like we lost installing the slack app 👍🏻  but I think I honestly prefer using the web version on linux, I'm more than ok with leaving it up to individual ubuntu users to determine if they want to install it themselves.